solvation_analysis/tests/test_solvation_analysis.py

```diff
@@ -77,28 +85,62 @@ def u_real_named(u_real):
 
 @pytest.fixture
 def atom_groups(u_real):
-    li_atoms = u_real.atoms.select
-    pf6_atoms = u_real.atoms.select
-    bn_atoms = u_real.atoms.select
-    fec_atoms = u_real.atoms.select_atoms()
+    li_atoms = u_real.atoms.select_atoms("type 22")
+    pf6_atoms = u_real.atoms.select_atoms("type 20").residues.atoms
+    bn_atoms = u_real.atoms.select_atoms("type 5").residues.atoms
+    fec_atoms = u_real.atoms.select_atoms("type 21").residues.atoms
     atom_groups = {"li": li_atoms, "pf6": pf6_atoms, "bn": bn_atoms, "fec": fec_atoms}
-
-
-def test_solvation_analysis_imported():
-    """Sample test, will always pass so long as import statement worked"""
-    assert "solvation_analysis" in sys.modules
+    return atom_groups
 
 
 def test_get_atom_group(u_real_named):
-    return
-
+    u = u_real_named
+    res_group = u.residues[1:5]
+    atom_group = u.atoms[1:5]
+    res = u.residues[1]
+    atom = u.atoms[1]
+    groups = [res_group, atom_group, res, atom]
+    for group in groups:
+        assert isinstance(get_atom_group(u, group), mda.core.groups.AtomGroup)
+
+
+def test_get_closest_n_mol(u_grid_1, u_real, atom_groups):
```

Comment:
I would split into two tests, one that uses `u_real` and one that uses `u_grid_1`. That way the point of failure is clear. Or if you don't use a fixture, no need to take it as an argument.

Reply:
Done, this is definitely a cleaner way to do it.